charts/handbrake/1.0.0/questions.yaml

```diff
@@ -0,0 +1,699 @@
+  - variable: service
+    group: "Services"
+    label: "Configure Service"
+    schema:
+      type: dict
+      attrs:
+        - variable: type
+          label: "Service type"
+          schema:
+            type: string
+            default: "ClusterIP"
+            enum:
+              - value: "NodePort"
+                description: "NodePort"
+              - value: "ClusterIP"
+                description: "ClusterIP"
+            show_subquestions_if: "NodePort"
+            subquestions:
+              - variable: port
+                label: "Port configuration"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: webuiport
+                      label: "container port"
+                      schema:
+                        type: int
+                        default: 5800     #web GUI
+                        editable: false
+                    - variable: nodePort
+                      label: "Node Port to expose for UI"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 36052
+                        required: true
+  - variable: appAdditionalServices
+    group: "Services"
+    label: "Configure additional services"
+    schema:
+      type: dict
+      attrs:
+        - variable: vnc
+          label: ""
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable TCP port for VNC connections"
+                schema:
+                  type: boolean
+                  default: true
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: port
+                      label: "Configuration"
+                      schema:
+                        type: dict
+                        attrs:
+                          - variable: vncport
+                            label: "container targetPort"
+                            schema:
+                              type: int
+                              default: 5900  #VNC
+                              required: false
+                              editable: false     
+                          - variable: nodevncPort
+                            label: "container port"
+                            schema:
+                              type: int
+                              default: 36053
+                              required: false       
+                    - variable: type
+                      label: "service type"
+                      schema:
+                        type: string
+                        default: "NodePort"
+                        enum:
+                          - value: "ClusterIP"
+                            description: "ClusterIP"
+                          - value: "NodePort"
+                            description: "NodePort"    
```

Comment:
This looks like an absolute mess.
Did you make this yourself? Because just about none of these settings are actually correct.

Reply:
its kinda mix and match and some tinkering with old and new additional services. Trying to figure this out now

Reply:
Actually the structure is copy pasta from qbittorent, but changed some variables to vnc for some reason

Reply:
old and new additional services?
appAdditional services is our own design, which you can copy-paste from other Apps (like the torrent ones).
There is no "old additional services" (Only unused code)

Reply:
ahh that explains things...
I think if you fix your indentation (I'm not sure thats correct) and remove VNC from all those places it should be alright!

I also think the section "label: "Enable TCP port for VNC connections" isn't needed. It should always be true and hidden. because if set to clusterIP it's only availeble on the cluster (if enabled inside the APP)

Reply:
This is copy pasta from qbit, changed only the ports, 
When set to ClusterIP it still shows the ports textboxes, which doesn't look right.
Am i missing something here?
```
  - variable: appAdditionalServices
    group: "Services"
    label: "Configure additional services"
    schema:
      type: dict
      attrs:
        - variable: tcp
          label: ""
          schema:
            type: dict
            attrs:
              - variable: enabled
                label: "Enable TCP port for Torrent Connections"
                schema:
                  type: boolean
                  default: true
                  show_subquestions_if: true
                  subquestions:
                    - variable: port
                      label: "Configuration"
                      schema:
                        type: dict
                        attrs:
                          - variable: targetPort
                            label: "container targetPort"
                            schema:
                              type: int
                              default: 5900   # VNC
                              required: false
                          - variable: port
                            label: "container port"
                            schema:
                              type: int
                              default: 51413
                              required: false             
                    - variable: type
                      label: "service type"
                      schema:
                        type: string
                        default: "ClusterIP"
                        enum:
                          - value: "ClusterIP"
                            description: "ClusterIP"
                          - value: "NodePort"
                            description: "NodePort"
```

Reply:
I'm kinda confused because both torrent downloaders have different ui

![qbit](https://user-images.githubusercontent.com/47820033/108629009-95952d80-7466-11eb-8282-577579dd4cb5.png)
![transm](https://user-images.githubusercontent.com/47820033/108629011-962dc400-7466-11eb-996b-0bcc885904a2.png)

Reply:
copy the settings from the normal service, but put them under : 
```
        - variable: tcp
          label: ""
          schema:
            type: dict
            attrs:
```

add the enable, but set it to `default: true` and `hidden: true`

Reply:
reported for qbittorrent as #169

Reply:
It works now, but UI is still weird, when set to ClusterIP still shows the port textboxes
And with enabled hidden, still shows the checkbox